src/mito2/src/manifest/action.rs

```diff
@@ -396,7 +408,7 @@ impl RegionMetaActionList {
         Self { actions }
     }
 
-    pub fn into_region_edit(self) -> RegionEdit {
+    pub fn into_region_edit(self) -> (Option<RegionChange>, RegionEdit) {
```

Comment:
The method name is no longer clear.

Reply:
[a49e010](https://github.com/GreptimeTeam/greptimedb/pull/7456/commits/a49e0109015248e46f2f44505f9a219648a5e383)